src/file.cpp

```diff
@@ -373,6 +386,15 @@ namespace bx
 			return true;
 		}
 
+		virtual void flush() override
+		{
+			if (m_open
+				&& 0 != m_fd)
+			{
+				crt0::flush(m_fd);
```

Comment:
Can't find where is `crt0.h`. In my understanding, user may needs to implement it by himself/herself?